tests/stubtest_allowlists/py3_common.txt

```diff
@@ -272,3 +272,437 @@ os.O_[A-Z_]+
+distutils.msvccompiler.get_build_version
+distutils.msvccompiler.normalize_and_reduce_paths
+distutils.msvccompiler.read_keys
+distutils.msvccompiler.read_values
+distutils.sysconfig.parse_config_h
+distutils.sysconfig.parse_makefile
+distutils.text_file.TextFile.default_options
+distutils.text_file.TextFile.error
+distutils.text_file.TextFile.gen_error
+distutils.unixccompiler.UnixCCompiler.compiler_type
+distutils.unixccompiler.UnixCCompiler.dylib_lib_extension
+distutils.unixccompiler.UnixCCompiler.dylib_lib_format
+distutils.unixccompiler.UnixCCompiler.executables
+distutils.unixccompiler.UnixCCompiler.obj_extension
+distutils.unixccompiler.UnixCCompiler.shared_lib_extension
+distutils.unixccompiler.UnixCCompiler.shared_lib_format
+distutils.unixccompiler.UnixCCompiler.src_extensions
+distutils.unixccompiler.UnixCCompiler.static_lib_extension
+distutils.unixccompiler.UnixCCompiler.static_lib_format
+distutils.unixccompiler.UnixCCompiler.xcode_stub_lib_extension
+distutils.unixccompiler.UnixCCompiler.xcode_stub_lib_format
+distutils.util.grok_environment_error
+email.base64mime
+email.charset
+email.contentmanager.get_and_fixup_unknown_message_content
+email.contentmanager.get_message_content
+email.contentmanager.get_non_text_content
+email.contentmanager.get_text_content
+email.contentmanager.set_bytes_content
+email.contentmanager.set_message_content
+email.contentmanager.set_text_content
+email.encoders
+email.errors
+email.feedparser
+email.generator
+email.header
+email.headerregistry.AddressHeader.init
+email.headerregistry.AddressHeader.max_count
+email.headerregistry.ContentDispositionHeader.init
+email.headerregistry.ContentTransferEncodingHeader.init
+email.headerregistry.ContentTransferEncodingHeader.max_count
+email.headerregistry.ContentTypeHeader.init
+email.headerregistry.DateHeader.init
+email.headerregistry.DateHeader.max_count
+email.headerregistry.MIMEVersionHeader.init
+email.headerregistry.MIMEVersionHeader.max_count
+email.headerregistry.ParameterizedMIMEHeader.init
+email.headerregistry.ParameterizedMIMEHeader.max_count
+email.headerregistry.UniqueAddressHeader.max_count
+email.headerregistry.UniqueDateHeader.max_count
+email.headerregistry.UniqueSingleAddressHeader.max_count
+email.headerregistry.UniqueUnstructuredHeader.max_count
+email.headerregistry.UnstructuredHeader.max_count
+email.iterators
+email.iterators.walk
+email.message
+email.message.Message.raw_items
+email.message.Message.set_raw
+email.mime
+email.parser
+email.policy.Compat32.mangle_from_
+email.policy.EmailPolicy.message_factory
+email.policy.Policy.mangle_from_
+email.policy.Policy.message_factory
+email.quoprimime
+email.utils
+hmac.HMAC.blocksize
+importlib.abc.Finder.find_module
+importlib.machinery.WindowsRegistryFinder.DEBUG_BUILD
+importlib.machinery.WindowsRegistryFinder.REGISTRY_KEY
+importlib.machinery.WindowsRegistryFinder.REGISTRY_KEY_DEBUG
+inspect.Signature.from_builtin
+inspect.Signature.from_function
```

Comment:
These functions are deprecated and have been removed in Python 3.11, so we might want to add a comment to that effect or put them in a "Won't fix" section